transcode/transcode.go

```diff
@@ -165,6 +156,92 @@ func RunTranscodeProcess(transcodeRequest TranscodeSegmentRequest, streamName st
 		return outputs, segmentsCount, err
 	}
 
+	// transmux received segments from T into a single mp4
+	if transcodeRequest.GenerateMP4 {
+		for rlist, slist := range renditionList.RenditionSegmentTable {
+			// create folder
+			if _, err := os.Stat(TRANSMUX_STORAGE_DIR); os.IsNotExist(err) {
+				os.MkdirAll(TRANSMUX_STORAGE_DIR, 0700)
+			}
```

Comment:
we could just omit the existence check since `MkdirAll` just does a noop if it already exists